src/Ds2013/Presenters/Section/Footer/FooterPresenter.php

```diff
@@ -32,11 +32,8 @@ public function getNetwork(): ?Network
 
     public function getNetworkUrlKey(): string
     {
-        // urlKey is nullable
-        if ($this->network && $this->network->getUrlKey()) {
-            return $this->network->getUrlKey();
-        }
-        return '';
+        $urlKey = $this->network ? $this->network->getUrlKey() : null;
```

Comment:
If you're making this into a ternary why not just do
```suggestion
        return $this->network ? $this->network->getUrlKey() : '';
```
?

Reply:
Actually, ignore that, I'm an idiot. I think this read better before though tbh.